galaxy/worker/logutils.py

```diff
@@ -29,8 +28,8 @@ def process(self, msg, kwargs):
 
 
 class ImportTaskAdapter(BaseLoggerAdapter):
-    def __init__(self, logger, task_id):
-        super(ImportTaskAdapter, self).__init__(logger, {'task_id': task_id})
+    def __init__(self, logger, task):
+        super(ImportTaskAdapter, self).__init__(logger, {'task': task})
```

Comment:
In Python 3 you can omit the arguments to `super` and just do
```         super().__init__(logger, {'task': task}) ```

see https://www.python.org/dev/peps/pep-3135/

Reply:
Sure, will update. Dropping python 2 compatible code was not a part of this PR. I may change it here and in other places as a separate PR or update this one.

Reply:
I'm a big fan of "as you go" updates and refactors, which is why I only said anything at all because this line was already being touched. Its not a big deal, just pointing it out in-case the change wasn't on your radar. Not a blocker.